package.json

```diff
@@ -51,7 +51,7 @@
   },
   "dependencies": {
     "debug": "^4.3.1",
-    "node-saml": "^4.0.0-beta.0",
+    "node-saml": "file:../node-saml",
```

Comment:
btw. is this change correct? there isn't `../node-saml` in the source tree (following link points to source tree of the [commit](https://github.com/node-saml/passport-saml/pull/619/commits/df81d0f5651a5e915e1fa27ce16c12122f33d8cd) which changed this line: https://github.com/node-saml/passport-saml/tree/df81d0f5651a5e915e1fa27ce16c12122f33d8cd/src )

Reply:
This change is only temporary in order to test changes. We won't land the code like this. When this is completely approved and ready to land, we'll land the `node-saml` part first, then publish it, and then point this to that.